flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/search/ConnectorSearchCommand.java

```diff
@@ -120,4 +140,40 @@ private static String getLastPackagePrefix(String rawPackageName) {
                 ? rawPackageName.substring(rawPackageName.lastIndexOf('.') + 1) : rawPackageName;
         return trimmedPackageName.substring(0, 1).toUpperCase(Locale.ROOT) + trimmedPackageName.substring(1);
     }
+
+    private  List<SearchResult> getLocalConnectors() {
+        PackageCompilation compilation = project.currentPackage().getCompilation();
+        Iterable<Module> modules = project.currentPackage().modules();
+        List<SearchResult> localConnections = new ArrayList<>();
+        for (Module module : modules) {
+            if (module.isDefaultModule()) {
+                continue;
```

Comment:
Can't we support the default module's connectors?

Reply:
Yes we can, I escaped this to consider generated clients only.